### PR TITLE
Build dependencies bare metal script

### PR DESCRIPTION
**Pull request by Mionsz**
Build dependencies bare metal script added.
Script prepares the environment for media_proxy build with all feature enabled for host machine.

### CODE REVIEW

**Comment by ko80** [comment]
Since it doesn't affect functionality, I don't mind merging it.

**Comment by Mionsz** [comment]
@ko80 please give an approve then :)

**Comment by ko80** [comment]
I'd suggest adding any brief description of the script added. It could be in readme or anywhere else.

File: docs/README.md [hunk 1]
```diff
@@ -59,8 +59,13 @@ Detailed information about MCM SDK can be found in [sdk](sdk) directory.
 
 ### Basic Installation
 
-1. **Install Dependencies**
+1. **Install Dependencies**, choose between options `a)` or `b)`.
 
+    a) The `build_dep.sh` script can be used for all-in-one environment preparation. The script is designed for debian and was tested under `Ubuntu 20.04`, `Ubuntu 22.04` and `Ubuntu 24.04` environments.
```
Comment (ko80) [inline docs/README.md:64]:
```suggestion
    a) The `build_dep.sh` script can be used for all-in-one environment preparation. The script is designed for Debian and was tested under `Ubuntu 20.04`, `Ubuntu 22.04` and `Ubuntu 24.04` environments.
```

File: docs/README.md [hunk 2]
```diff
@@ -59,8 +59,13 @@ Detailed information about MCM SDK can be found in [sdk](sdk) directory.
...
+   To use this option just run the script after cloning the repository by typing `build_dep.sh`
+  
+    b) The following method is universal and should work for most Linux OS.
```
Comment (ko80) [inline docs/README.md:67]:
```suggestion
    b) The following method is universal and should work for most Linux OSes.
```

File: docs/README.md [hunk 3]
```diff
@@ -79,22 +84,22 @@ Detailed information about MCM SDK can be found in [sdk](sdk) directory.
     ```
     - Reboot
 
-2. **Clone the repository**
+3. **Clone the repository**
```
Comment (ko80) [inline docs/README.md:87]: Something happened to the numbering. Restart here with 2

**Review by ko80 — APPROVED** [review]
Looks good

**Pull request merged**